client/src/components/atoms/style/ScrollWindow.scss

```diff
@@ -1,6 +1,6 @@
 .scroll-window {
   width: 100%;
-  height: 100%;
+  max-height: 100%;
```

Comment:
Left some comments in Searchable Tag Dropdown https://github.com/uwblueprint/pregnancy-centre/pull/126
about this and other stuff. I guess please address in both PRs